packages/augur-core/source/contracts/libraries/BytesToString.sol

```diff
@@ -0,0 +1,30 @@
+
+pragma solidity 0.5.10;
+
+
+/**
+ * @title BytesToString
+ * @dev Bytes operations to convert to string and remove whitespace
```

Comment:
Removes nulls rather than whitespace.

Reply:
I'm not sure why you're filtering out nulls but not other non-printable characters. Just to conform to solidity's (evm's?) string encoding? It'd make sense if it were null-terminated.

Reply:
Because we store strings as bytes32s in this context the strings are padded to the right with a bunch of null characters

Reply:
If they're always to the right then you could get the length by reading until you hit a null. That also simplifies the rest of this function.

Reply:
Someone could always add a null char to the actual string

Reply:
We may need to restrict that. Someone could construct a market where two of the outcomes have an identical REP symbol by doing something like `so\0 happy` and `so\0 sad`.

Reply:
Those would end up being `sohappy` and `sosad` as is. For them to be identical theyd have to only differ by number of null chars

Reply:
Which is edge enough that I'm inclined to not care